catalog/views.py

```diff
@@ -1,3 +1,26 @@
+from itertools import product
+from venv import create
 from django.shortcuts import render
+from django.template import context
+from django.utils.translation import gettext as _
+from .models import *
+from django.shortcuts import get_object_or_404
 
-# Create your views here.
+def store(request):
+    products =Product.objects.all()
```

Comment:
Thêm xử lý sắp xếp sản phẩm theo tiêu chí nào đó đi em
vd: sản phẩm mới nhất chẳng hạn
```suggestion
    products = Product.objects.all()
```